### DISCUSSION TITLE
Androgynous voice(s)?

### DISCUSSION THREAD

**Discussion by caniplzjustlogin4godsakesthistakesforev** [rhasspy/piper#596] [Q&A]
Just wondering if there is/will be support for androgynous/ gender neutral voice profiles like apple's Quinn for siri

**Comment by luiscarlos2000** [comment]
There could be, as long as you have the datasets for the voice you want to train on. I believe it'l take a lot of time to train it depending on the GPU and such, but maybe could be worth it
